packages/remirror__react-renderer/src/handlers/link.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC } from 'react';
+import { Literal } from '@remirror/core';
+
+export const createLinkHandler = (overwriteProps?: Record<string, Literal>) => {
```

Comment:
@whawker When you implemented the static renderer: Did you also have to overwrite attrs (e.g. to open links in new tab)? If so: Which pattern did you use to configure the node renderer?

Reply:
Yes, I did have cause to overwrite in some cases, the pattern I used was similar to the handler map you have [here](https://github.com/remirror/remirror/blob/479fad906aebcf2168249ee65e0553543b6c9e19/packages/remirror__react-renderer/src/renderer.tsx#L24-L36)

If I required custom logic for a node type or mark, it renders a React component as a handler to support that, instead of rendering the default HTML node.

Does that answer the question, or have I missed something?

Reply:
Thanks a lot. Very helpful!